plugins/scaffolder-backend-module-gitlab/report.api.md

```diff
@@ -9,104 +9,6 @@ import { JsonObject } from '@backstage/types';
 import { ScmIntegrationRegistry } from '@backstage/integration';
 import { TemplateAction } from '@backstage/plugin-scaffolder-node';
 
-// @public
-export const createGitlabGroupEnsureExistsAction: (options: {
```

Comment:
I don't think we want to remove the existing exports as this would be a breaking change that we're not ready for yet.